tests/test_api_request_handler_labels.py

```
@@ -0,0 +1,1075 @@
+            status_code=200,
+            response_text={
+                "offset": 0,
+                "limit": 250,
+                "size": 1,
+                "labels": [{"id": 1, "title": "Label 1"}]
+            },
+            error_message=None
+        )
+        
+        with patch.object(labels_handler.client, 'send_get', return_value=mock_response):
+            result, error = labels_handler.get_labels(project_id=1, offset=0, limit=250)
```

Ruff (0.14.3)

Unpacked variable `result` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)